### PR TITLE
FOGL-711 Fixed error return from browser code

### PR DESCRIPTION
**Pull request by MarkRiddoch**

### CODE REVIEW

File: python/foglamp/services/core/api/browser.py [hunk 1]
```diff
@@ -82,7 +82,10 @@ def setup(app):
     _storage = connect.get_storage()
     results = _storage.query_tbl_with_payload('readings', payload)
 
-    return web.json_response(results['rows'])
+    if 'rows' in results:
+        return web.json_response(results['rows'])
+    else:
+        raise web.HTTPBadRequest(reason=results['message'])
```
Comment (ashish-jabble) [inline python/foglamp/services/core/api/browser.py]: Its NOT the case of `Bad Request error` but 200 with No data available for non-numeric readings. We may handle like below:

```
if 'rows' not in results:
     response = results['message']
else:
    response = results['rows']

return web.json_response({reading: response})
```

File: python/requirements.txt [hunk 2]
```diff
@@ -2,6 +2,9 @@
 aiocoap==0.3
 cbor2==4.0.0
 
+# Postgres for backup
+psycopg2==2.7.1
+
```
Comment (ashish-jabble) [inline python/requirements.txt]: Assuming this addition because we are using it for backup restore process. So all good 👍

Comment (MarkRiddoch) [reply python/requirements.txt]: Correct, a little naughty to sneak it in with this change.

File: python/foglamp/services/core/api/browser.py (hunk 1 above)
Comment (MarkRiddoch) [reply python/foglamp/services/core/api/browser.py]: We could, or as it currently is it says Bad Request with the message saying the data type is not compatible with the request. I'm not sure we should give 200 OK unless we return real values.

**Pull request merged**